docs/measurements/ripple.md

```diff
@@ -0,0 +1,18 @@
+# Ripple
+This service performs Ripple measurement.
+
+## Hardware Setup
+
+![alt text](meas-images/ripple-hardware-setup.png)
+
+## InstrumentStudio Panel
+
+### Usage
+
+1. Select the appropriate source, load and scope resource names and update other parameters as needed. Please note that, the measurement is in 'Perform Measurement' mode of operation by default.
+
+   ![alt text](meas-images/ripple-config.png)
+
+2. Run the measurement. The Ripple Voltage values are displayed on the graph. The RMS and Peak-to-Peak values of ripple are calculated and displayed in the table along with voltage and current values of source and load devices.
```

Comment:
please remove table as in python we don't have it.

Reply:
removed table and made it as indicators.